src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  line-height: 60px;
```

Comment:
It is a bad idea. You can just delete it
![image](https://user-images.githubusercontent.com/98764641/197800598-334a47c6-13de-4e3f-9fb2-366577683473.png)